spec/factories/async_transactions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :async_transaction do
+    sequence(:id, 1) { |n| n }
```

Comment:
Can you remove this id sequencing, b/c we'll want to allow FactoryBot to create unique ids for each record it creates.  Also, we'll want to define the class for the base factory.

I just tried this in the rails console, and kept getting unique constraint violations.  I fiddled with it a bit, and here is the resulting working file:

```ruby
# frozen_string_literal: true

FactoryBot.define do
  factory :async_transaction, class: AsyncTransaction::Base do
    sequence(:user_uuid, 100) { |n| "x3fab2b5-6af0-45e1-a9e2-394347af9#{n}" }
    sequence(:source_id, &:to_s)
    source 'vet360'
    status 'started'
    sequence(:transaction_id, 100) { |n| "d2fab2b5-6af0-45e1-a9e2-394347af9#{n}" }
    transaction_status 'RECEIVED'

    factory :address_transaction, class: AsyncTransaction::Vet360::AddressTransaction do
    end

    factory :email_transaction, class: AsyncTransaction::Vet360::EmailTransaction do
    end

    factory :telephone_transaction, class: AsyncTransaction::Vet360::TelephoneTransaction do
    end
  end
end
```